spec/prog/minio/minio_pool_nexus_spec.rb

```diff
@@ -65,27 +65,22 @@
 
   describe "#wait_servers" do
     it "waits if nothing to do" do
-      st = instance_double(Strand, label: "wait_servers")
-      ms = instance_double(MinioServer, strand: st)
-      expect(nx.minio_pool).to receive(:servers).and_return([ms])
+      # Server strand starts at "start", not "wait" - so it waits
       expect { nx.wait_servers }.to nap(5)
     end
 
     it "hops to wait if all servers are waiting" do
-      st = instance_double(Strand, label: "wait")
-      ms = instance_double(MinioServer, strand: st)
-      expect(nx.minio_pool).to receive(:servers).and_return([ms])
+      nx.minio_pool.servers.each { it.strand.update(label: "wait") }
       expect { nx.wait_servers }.to hop("wait")
     end
 
     it "triggers reconfigure if addition_pool_set" do
-      st = instance_double(Strand, label: "wait")
-      ms = instance_double(MinioServer, strand: st)
-      expect(nx.minio_pool).to receive(:servers).and_return([ms])
+      minio_cluster_strand  # ensure strand exists for semaphore
+      nx.minio_pool.servers.each { it.strand.update(label: "wait") }
       expect(nx).to receive(:when_add_additional_pool_set?).and_yield
-      expect(nx.minio_pool.cluster).to receive(:incr_reconfigure)
       expect(nx).to receive(:decr_add_additional_pool)
```

Comment:
Ideally for these 2 remaining existing lines:

1. Create the semaphore in the database
2. Create the semaphore in the database, and check after method that it no longer exists